pytorch_lightning/trainer/trainer.py

```diff
@@ -566,7 +531,45 @@ def _set_wide_running_stage(self, stage):
 
         self._running_stage = stage
 
+    def pre_training_routine(self):
```

Comment:
shall it be protected as it is not meant to be user api
```suggestion
    def _pre_training_routine(self):
```

Reply:
Yes.